### DISCUSSION TITLE
Jogosultság

### DISCUSSION THREAD

**Discussion by laszlo90** [nav-gov-hu/eVAT#69] [Kérdések és válaszok - Gépi interfész]
Üdvözlöm!
queryDocumentList lekérdezés esetében ez a válasz jön vissza.
Jogosultság szükséges! (FORBIDDEN)"
Online számla teszt rendszerben létrehoztuk a technikai felhasználót és bejelöltük az eáfa hozzáférést. Mi lehet a probléma?

**Comment by szako** [comment]
Az eÁFA webfelületen is engedélyezni kell a felhasználókat.

**Comment by renced42** [comment]
Kedves @laszlo90 

Ez nem szakmai kérdés, kérlek, hogy ha még fenn áll a hiba akkor Issueként jelentsd be.
Köszönöm szépen!
